src/test/java/co/rsk/bitcoinj/script/RedeemScriptParserFactoryTest.java

```diff
@@ -84,18 +80,16 @@ public void get_whenFlyoverStandardRedeemScript_shouldReturnRedeemScriptParser()
 
         RedeemScriptParser redeemScriptParser = RedeemScriptParserFactory.get(flyoverRedeemScript.getChunks());
 
-        Assert.assertTrue(redeemScriptParser instanceof FlyoverRedeemScriptParser);
-        assertInternalRedeemScriptParser((FlyoverRedeemScriptParser) redeemScriptParser, StandardRedeemScriptParser.class, STANDARD_MULTISIG);
-        Assert.assertEquals(MultiSigType.FLYOVER, redeemScriptParser.getMultiSigType());
+        assertTrue(redeemScriptParser instanceof FlyoverRedeemScriptParser);
+        assertInternalRedeemScriptParser((FlyoverRedeemScriptParser) redeemScriptParser, StandardRedeemScriptParser.class);
     }
 
-    private void assertInternalRedeemScriptParser(FlyoverRedeemScriptParser flyoverRedeemScriptParser, Class<?> expectedInternalRedeemScripParser, MultiSigType expectedType) {
+    private void assertInternalRedeemScriptParser(FlyoverRedeemScriptParser flyoverRedeemScriptParser, Class<?> expectedInternalRedeemScripParser) {
```

Comment:
I think we discussed this some time ago. Not a fan of testing the internals of a class. I prefer to focus on testing behavior.

Assert that for a given script we receive certain responses from the obtained parser. That's what we care about

Reply:
Updated